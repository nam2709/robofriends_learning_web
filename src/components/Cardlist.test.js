import React from 'react';
import { render } from '@testing-library/react';
import Cardlist from './Cardlist';

// Ensure that Jest knows how to handle .snap files
import '@testing-library/jest-dom/extend-expect';

test('MyComponent snapshot', () => {
  const mockRobots = [ 
    {
        id: 1,
        name: 'robot test',
        email: 'robottest@gmail.com'
    }
  ]
  const { asFragment } = render(<Cardlist robots={mockRobots}/>);
  expect(asFragment()).toMatchSnapshot();
});