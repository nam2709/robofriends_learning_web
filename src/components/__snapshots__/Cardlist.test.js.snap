// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
    >
      <img
        alt="robots"
        src="https://robohash.org/1?size=200x200"
      />
      <div>
        <h2>
           robot test 
        </h2>
        <p>
           robottest@gmail.com 
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
